app/public/privacy/privacy.html

```diff
@@ -1,18 +1,78 @@
 <!DOCTYPE html>
-<html lang='en'>
+<html>
 <head>
-	<meta charset='UTF-8'>
-	<meta name='viewport' content='width=device-width, initial-scale=1.0'>
-	<title>Placeholder for Privacy Statement / Policy, AUP</title>
-	<style type='text/css'>body{margin:40px auto;max-width:650px;line-height:1.6;font-size:18px;color:#444;padding:0 10px}h1,h2,h3{line-height:1.2}</style>
 </head>
 <body>
-	<header><h1>Privacy Statement</h1></header>
-	<h2>Privacy Policy</h2>
-	<ul>
-		<li>User consent</li>
-		<li>Data deletion</li>
-	</ul>
-	<h2>Acceptable use policy (AUP)</h2>
+<h1>eduMeet &ndash; Privacy Notice</h1>
```

Comment:
This is not the proper place for edumeet specific instance privacy notice. Geant is not responsible for instances forked from this repository, nor do we want to be contacted by the users of various instances. This needs to only be a skeleton, please revert.